src/pyvesync/vesync.py

```diff
@@ -98,17 +161,18 @@ def process_devices(self, devices) -> tuple:
                 # elif devType in bulb_types:
                 #    bulbs.append(VSFactory.getDevice(devType, dev, self))
                 else:
-                    logger.debug('Unknown device ' + devType)
+                    logger.error('Unknown device ' + devType)
```

Comment:
is there a reason to switch off of debug to error? error statements are logged in HA by default so we might be adding noise to the logs unless there is a good reason

Reply:
The logic behind this is if someone buys a new device and it's not supported, it will show up in the logs to indicate why the device isn't appearing.  Basically to skip a step in troubleshooting if someone raises an issue with an unsupported device.  I was on the fence with this.  What do you think?